pkg/db/db_test.go

```diff
@@ -0,0 +1,91 @@
+package db
+
+import (
+	"os"
+	"testing"
+)
+
+const testDbPath string = ":memory:"
+
+var (
+	testConn    *Database
+	testConnErr error
+)
+
+func TestMain(m *testing.M) {
+	setup()
+	c := m.Run()
+	teardown()
+	os.Exit(c)
+}
+
+func setup() {
+	testConn = connect(testDbPath)
+	testConn.CreateTable()
```

Comment:
Instead of using CreateTable just execute it in the test directly and I don't think that the testDbPath constant is necessary in that case, or is it?